api/src/wfl/module/wgs.clj

```diff
@@ -41,31 +41,30 @@
 
 (def cram-ref
   "Ref Fasta for CRAM."
-  {:cram_ref_fasta        "gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.fasta"
-   :cram_ref_fasta_index  "gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.fasta.fai"})
+  (let [hg38           "gs://gcp-public-data--broad-references/hg38/v0/"
+        cram_ref_fasta (str hg38 "Homo_sapiens_assembly38.fasta")]
+    {:cram_ref_fasta        cram_ref_fasta
+     :cram_ref_fasta_index  (str cram_ref_fasta ".fai")}))
```

Comment:
2 parts of me have been struggling on "explicit hard-coded consts" v.s. "dynamically generate data" in Clojure for a year, I preferred the former simply because it's more friendly for full text search, but I guess this (latter) is a de facto in clj?

Reply:
I'd prefer these were just literals too now, but the code isn't consistent.

Wanted to just factor the data, but discovered wgs.clj now has a synthetic
`reference_fasta_prefix` parameter that necessitates something like a
`(defn reference_fasta [prefix] …)` to support it.  Tackling that in this PR
is a design change (and definitely a bridge too far).

Reply:
That said, for things like `foo` and `foo.index`, I think we should do something to emphasize that the index is just the root with a suffix added.